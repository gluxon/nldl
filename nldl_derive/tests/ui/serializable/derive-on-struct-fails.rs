#[derive(Debug, PartialEq, nldl::attr::Deserialize)]
struct TestStruct {}

fn main() {}
