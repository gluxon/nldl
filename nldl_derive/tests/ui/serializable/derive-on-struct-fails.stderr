error: proc-macro derive panicked
 --> tests/ui/serializable/derive-on-struct-fails.rs:1:28
  |
1 | #[derive(Debug, PartialEq, nldl::attr::Deserialize)]
  |                            ^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: NetlinkAttributeDeserializable derive may only be used on enums.
